src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectOptionsProcessor.cs

```diff
@@ -10,24 +13,24 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
     internal class VisualStudioProjectOptionsProcessor : IDisposable
     {
         private readonly VisualStudioProject _project;
-        private readonly HostWorkspaceServices _workspaceServices;
+        private readonly VisualStudioWorkspaceImpl _workspace;
         private readonly ICommandLineParserService _commandLineParserService;
 
         /// <summary>
         /// Gate to guard all mutable fields in this class.
         /// The lock hierarchy means you are allowed to call out of this class and into <see cref="_project"/> while holding the lock.
         /// </summary>
-        private object _gate = new object();
+        private readonly object _gate = new object();
```

Comment:
Oh goodness thank you. :smile:

Reply:
Thank the analyzer that squiggled this :)